tests/auto/ci_auto.py

```diff
@@ -119,18 +122,21 @@ class Job:
         provided by the bash script
     '''
 
-    def __init__(self, preq_dict, ghinterface_obj, machine, compiler):
+    def __init__(self, preq_dict, ghinterface_obj, machine, compiler,
+                 repo, hpc_acc):
         self.logger = logging.getLogger('JOB')
         self.preq_dict = preq_dict
-        self.job_mod = importlib.import_module(
-                       f'jobs.{self.preq_dict["action"].lower()}')
+        # both build and WE2E tests call same module
+        self.job_mod = importlib.import_module('jobs.build')
```

Comment:
Just curious, will this class always be used to build?

Reply:
This is part of the code that I brought over from the model testing. They had 2 program options - one for baseline tests (jobs/bl.py) and one for regression (jobs/rt.py). I wasn't sure if we would need to do something like that later, so I minimally changed it. As the code now stands, only jobs/build.py is imported this way. Did that answer your question?